src/simulator/publisher.rs

```diff
@@ -20,17 +20,122 @@ use crate::{bench::ConnectionError, simulator::PubStats, SimulatorConfig};
 struct Imu {
     sequence: u32,
     timestamp: u64,
+    #[dummy(faker = "1.0 .. 2.8")]
     ax: f64,
+    #[dummy(faker = "1.0 .. 2.8")]
     ay: f64,
+    #[dummy(faker = "9.79 .. 9.82")]
     az: f64,
+    #[dummy(faker = "0.8 .. 1.0")]
     pitch: f64,
+    #[dummy(faker = "0.8 .. 1.0")]
     roll: f64,
+    #[dummy(faker = "0.8 .. 1.0")]
     yaw: f64,
+    #[dummy(faker = "-45.0 .. -15.0")]
     magx: f64,
+    #[dummy(faker = "-45.0 .. -15.0")]
     magy: f64,
+    #[dummy(faker = "-45.0 .. -15.0")]
     magz: f64,
 }
 
+#[derive(Debug, Serialize, Dummy)]
+pub struct Location {
```

Comment:
nit: we can name this `Gps` to be consistent with other places we are referring to this type of data in code